lib/opts/index.ts

```diff
@@ -0,0 +1,122 @@
+import { getConfiguration as getConfigFile } from "../configuration";
+import { isin } from "../../type-helpers";
+import type { PrettyPrint } from "../../type-helpers";
+
+type Release = "minor" | "major" | "patch";
+type Task = "changelog" | "commit" | "tag";
+type Hook =
+  | "prerelease"
+  | "prebump"
+  | "postbump"
+  | "prechangelog"
+  | "postchangelog"
+  | "precommit"
+  | "postcommit"
+  | "pretag"
+  | "posttag";
+
+type TypePrefixes = Array<
+  (
+    | { section: string; hidden?: boolean | undefined }
+    | { hidden: true; section?: string | undefined }
+  ) & { type: string }
+>;
+
+type ConfigFiles = Array<
+  | string
+  | { filename: string; type: "json" | "gradle" | "plain-text" }
+  | { filename: string; updater: string }
+>;
+
+/**
+ * __THIS SHOULD NOT CHANGE.__
+ * @deprecated
+ *
+ * The configuration options for `standard-version` as of version 9.5 (The last version prior to the fork; deprecated).
+ */
+export type SVConfig = {
+  packageFiles: ConfigFiles;
+  bumpFiles: ConfigFiles;
+  releaseAs: Release;
+  prerelease: string | boolean;
+  infile: string;
+  message: string;
+  firstRelease: boolean;
+  sign: boolean;
+  noVerify: boolean;
+  commitAll: boolean;
+  silent: boolean;
+  tagPrefix: string;
+  scripts: Record<Hook, string>;
+  skip: Record<Task, string>;
+  dryRun: boolean;
+  gitTagFallback: boolean;
+  path: string;
+  changelogHeader: string;
+  preset: string;
+  lernaPackage: string;
+  header: string;
+  types: TypePrefixes;
+  preMajor: boolean;
+  commitUrlFormat: string;
+  compareUrlFormat: string;
+  issueUrlFormat: string;
+  userUrlFormat: string;
+  releaseCommitMessageFormat: string;
+  issuePrefixes: string[];
+};
+
+/**
+ * The configuration object for commit-and-tag-version, which is a superset of the conventional-changelog-config-spec (as of version 2.1.0)
+ * This may or may not maintain backwards compatibility with standard-version (as of version 9.5.0).
+ */
+export type CatVConfig = PrettyPrint<
```

Comment:
Similarly, you could call this `Config` unless you have a good reason not to.

One thing that I didn't like about the new package name I chose was that it doesn't have an easy abbreviation 😬

Reply:
Lol yeah, CatV (or some other capitalization) is the best I could think of. This can be renamed, though at the loss of being explicit about this config being parallel, but divergent from, StandardVersionConfig (although, again, it is mentioned in the tooltip.)